src/mf-select.component.ts

```diff
@@ -225,20 +224,27 @@ export class MfSelectComponent implements OnInit, AfterViewInit, OnChanges, OnDe
     switch ($event.code) {
       case 'ArrowDown':
         this.open();
-        this.markedItem = this.markedItem < this.filteredItems.length - 1 ? this.markedItem + 1 : this.markedItem;
-        this.virtualScrollComponent.scrollInto(this.filteredItems[this.markedItem]);
+        this.markedItem = this.findNextNonCategoryItem(this.markedItem !== undefined ? this.markedItem + 1 : 0);
```

Comment:
What happens if you have 3 items with the middle item being disabled and the first item marked, but then you press the down key? Does it select nothing, or would it skip over the disabled item and select the next available item?